provision/infinispan/README.adoc

```diff
@@ -53,6 +53,28 @@ CLUSTER_1="gh-pruivo" CLUSTER_2="gh-keycloak" NS_1="ispn-server" NS_2="ispn-serv
 CLUSTER_1="gh-pruivo" CLUSTER_2="gh-keycloak" NS_1="server-site-1" NS_2="server-site-2" ./create_ispn_clusters.sh
 ----
 
+== Taskfile
+
+For single cluster mode:
```

Comment:
We should add corresponding delete tasks for each of these tasks.

Reply:
+1 I am not currently sure, what is the best way to delete the ISPN clustered configuration., would I just delete the `Infinispan` namespace(s) and reset the config in the Keycloak deployment `.env` ?

Reply:
Added a task to delete the Infinispan CR. Didn't delete the namespace because I've been running keycloak and infinispan in the same namespace.